api/controllers/CheckUploadedDocumentsController.js

```diff
@@ -8,6 +8,10 @@ const addUserDataToDB = require('../helper/addUserDataToDB.js');
 
 const CheckUploadedDocumentsController = {
     async renderPage(req, res) {
+
+        if (HelperService.maxFileLimitExceeded(req))
+            return res.serverError('maxFileLimitExceeded');
```

Comment:
For this specific scenario no, it renders a server error page purely because a user would have to essentially hack the flow to get this error.

Also, Liz is a bit swamped at the moment and wouldn't have any time to write copy for an error page that tells the user to go back and fix the issue they ignored on the upload page 😁

Reply:
No worries.  One to add to the backlog as it should ideally display a sensible error message.